protos/protos/public/modeldb/DatasetVersionService.proto

```diff
@@ -115,6 +115,7 @@ message GetAllDatasetVersionsByDatasetId {
 
 message DeleteDatasetVersion {
     string id = 1;
+    string dataset_id = 2;
```

Comment:
@ad-47 : is this optional argument? we should make it optional and look up the repo and error out if the commit belongs to multiple dataset repos .

Reply:
same for the other APIs in here.